python/fedml/api/modules/storage.py

```diff
@@ -474,8 +491,11 @@ def _get_storage_service(service):
     else:
         raise NotImplementedError(f"Service {service} not implemented")
 
-def _get_data_type(data_path):
-    if os.path.isdir(data_path):
+
+def _get_data_type(data_path, byte_data_flag):
+    if byte_data_flag:
+        return DataType.BYTE
```

Comment:
Is there not a way to determine whether input data is of byte type or not without this flag?